staging/src/k8s.io/apiserver/pkg/storage/storagebackend/config.go

```diff
@@ -45,6 +48,8 @@ type Config struct {
 
 	Codec  runtime.Codec
 	Copier runtime.ObjectCopier
+	// Transformer allows the value to be transformed prior to persisting into etcd.
+	Transformer value.Transformer
```

Comment:
How do use the new transformers? Will additional flags be added to https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/server/options/etcd.go#L65 in a follow up PR?